app/models/cycle_type.rb

```diff
@@ -1,9 +1,48 @@
-class CycleType < ActiveRecord::Base # Defines things like unicycles and recumbent
-  include FriendlySlugFindable
+class CycleType
+  include Enumable
 
-  has_many :bikes
+  SLUGS = {
```

Comment:
Is there a reasonable way you can think of combining this `SLUGS` hash with the `NAMES` hash?

Reply:
@sethherr tbh I'm not sure it would be worth the added complexity. It would resemble something like:

```

# CycleType:

  MAPPING = [
    { slug: :cargo, name: "Cargo Bike (front storage)", enum: 0 },
    ...

# Enumable:

  class_methods do
    %w( names slugs ).each do |kind|
      define_method(kind.to_sym) do
        key = kind.singularize.to_sym;
        self::MAPPING.map{ |m| m.fetch(key).to_s }
      end
    end

    def enums
      Hash[self::MAPPING.map { |m| [m[:slug], m[:enum]] }]
    end

    def select_options
      self::MAPPING.map { |m| [m[:name], m[:slug]] }
    end

    def legacy_selections
      self::MAPPING.map{ |m| m.slice(:name, :slug).stringify_keys }
    end

# Bike:

enum cycle_type: CycleType.enums
```

Down for whatever though!

Reply:
yeah. I can't think of anything super smooth either. Let's keep it the way it is 🙂